lib/http/form_data/multipart.rb

```diff
@@ -24,7 +26,7 @@ def initialize(data)
       #
       # @return [String]
       def content_type
-        "multipart/form-data; boundary=#{@boundary}"
+        "multipart/form-data; boundary=#{boundary}"
```

Comment:
Ok, the only reason why I changed from `@boundary` to `boundary` here was because the reader method detects typos, but I can revert back.